src/app/login/success/page.tsx

```
@@ -33,8 +34,14 @@ const SuccessPage = () => {
           case 'ROLE_ADMIN':
             router.push('/admin');
             break;
+          case 'ROLE_USER':
           default:
-            router.push('/');
+            const isOnboardingCompleted = onboardingUtils.isCompleted();
+            if (isOnboardingCompleted) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)